examples/banananation/src/main/java/gov/nasa/jpl/aerie/banananation/activities/RelaxActivity.java

```diff
@@ -0,0 +1,19 @@
+package gov.nasa.jpl.aerie.banananation.activities;
+
+import gov.nasa.jpl.aerie.banananation.Mission;
+import gov.nasa.jpl.aerie.merlin.framework.annotations.ActivityType;
+import gov.nasa.jpl.aerie.merlin.framework.annotations.ActivityType.EffectModel;
+
+/**
+ * Relax? I needed something with no parameters this was the best one I could come up with
+ *
+ * @subsystem fruit
+ * @contact Jane Doe
```

Comment:
Pedantic since these `@` javadoc lines are harmless but I don't see a compelling reason to check them into VCS

Reply:
Also, this could be called `EffectlessActivity` if you're looking for another name

Reply:
I agree the name could be more descriptive, preferably something banana / monke themed...

Reply:
So should I just remove the @'s? It seems like all the other activities have them so I added them for the sake of uniformity. I can rename the activity to something better too.

Reply:
Ahh true! Didn't realize the others do. Nice, in that case I think what you have is the right thing to do. :)

Reply:
FYI, the *reason* those nonstandard Javadoc tags are present is because we originally (and, I think, still?) wanted to extract that information out at annotation processing time, so it could be flowed out to planners via the UI.

That hasn't happened yet, so these tags don't do anything. Their presence is a good reminder that we still have yet to implement that functionality... but only, of course, if we knew it was a thing to begin with 😁

Reply:
How's BananaNapActivity